systemtest/cmd/apmbench/main.go

```diff
@@ -49,6 +49,8 @@ func Benchmark1000Transactions(b *testing.B, l *rate.Limiter) {
 }
 
 func BenchmarkOTLPTraces(b *testing.B, l *rate.Limiter) {
+	b.Skipf("Disabled until we've addressed https://github.com/elastic/apm-server/issues/9242")
```

Comment:
Rather than disabling it right here disallowing us from running it on demmand, would it be best to update the `BENCHMARK_RUN` expression on the benchmarks so that the test isn't run? WDYT?

Reply:
Good point!

I think we should be able to run it on demand but I'm not sure about changing the benchmark statement. I went with a different approach and used the same code we are using for integration tests. Let me know what you think!